tokenizers/src/pre_tokenizers/metaspace.rs

```diff
@@ -55,6 +92,14 @@ impl Metaspace {
         self.replacement = replacement;
         self.str_rep = replacement.to_string();
     }
+
+    pub fn get_prepend_scheme(&self) -> PrependScheme {
+        self.prepend_scheme.clone()
```

Comment:
Don't clone on behalf of users, never.

Either return a reference `&PrependScheme`.
Or make `PrependScheme`  ` Copy`.

Making something Copy, is something you should do only when the size makes it worthwile.
A reference is a pointer of size `usize` so copying usize is usually much faster than copying an entire struct.
For an enum like PrependScheme, it' s only 3 possible values, encoded internally by rust as `u8` or `usize` (Don't remember which). In any case, it's prefereable to copy the value than to pass references around (which cost a pointer dereference)